contrib/babelfishpg_common/src/spatialtypes.c

```diff
@@ -0,0 +1,197 @@
+/*-------------------------------------------------------------------------
+ *
+ * spatialtypes.c
+ *    Functions for the type "geometry" and "geography".
+ *
+ *-------------------------------------------------------------------------
+ */
+#ifdef ENABLE_SPATIAL_TYPES
+
+#include "postgres.h"
+#include "fmgr.h"
+#include "utils/geo_decls.h"
+#include "utils/builtins.h"
+
+static void load_functions();
+
+/*
+ * Macros for identifying Z and M flags
+ */
+#define FLAG_Z         0x01
+#define FLAG_M         0x02
+#define FLAG_ZM        0x03
```

Comment:
These could be defined as:
```
#define FLAG_Z         1 << 0
#define FLAG_M        1 << 1
```
No need to define FLAG_ZM, it can be calculated as using above two as: (FLAG_Z | FLAG_M)

Reply:
Done